docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -226,3 +229,29 @@ and a file system location of ``/var/my_tables/test_table``::
         format = 'PARQUET',
         partitioning = ARRAY['c1', 'c2'],
         location = '/var/my_tables/test_table')
+
+.. _iceberg-materialized-views:
+
+Materialized views
+^^^^^^^^^^^^^^^^^^
+
+The Iceberg connector supports :ref:`sql-materialized-views-management`. In the
+underlying system each materialized view consists of a view definition and an
+Iceberg storage table. The storage table name is stored as a materialized view
+property. The data is stored in that storage table.
+
+You can use the :ref:`iceberg-table-properties` to control the created storage
+table and therefore the layout and performance. For example, you can use the
+following clause with :doc:`/sql/create-materialized-view` to use the ORC format
+for the data files and partition the storage per day using the column
+``_date``::
+
+    WITH ( format = 'ORC', partitioning = ARRAY['_date'] )
```

Comment:
leading underscores are not typical. Let's use `a_date`, `some_date` or `event_date`

Reply:
done